.travis.yml

```diff
@@ -23,6 +23,21 @@ matrix:
   allow_failures:
     - env: lt_branch=RC_1_0 gui=true build_system=cmake
     - env: lt_branch=RC_1_0 gui=false build_system=cmake
+  include:
+    - language: python
+      python: 2.7   # legacy Python
+      addons: true  # override
+      before_install: true
+      env: build_system=Python
+      install: pip install flake8
+      script: flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+    - language: python
+      python: 3.7   # Python
```

Comment:
minimum requirement is 3.3, please change it.

Reply:
Flake8 no longer runs on Python 3.3 was end of life years ago and Python 3.4 will be in end of life in the coming weeks.  3.7 has a stricter syntax that previous versions of Python 3 so it makes a good proxy for the rest.